pyntc/devices/ios_device.py

```diff
@@ -69,6 +69,18 @@ def _raw_version_data(self):
             return version_data
         except IndexError:
             return {}
+            
+    def _reconnect(self, timeout=60):
+        counter = 0
+        timeout = timeout*4
```

Comment:
The timeout should be a second counter, and then use time library to measure length of time.